source/main.coffee

```diff
@@ -78,6 +78,7 @@ uncacheable = new Set [
   "NestedJSXChildExpression"
   "NestedObject"
   "NestedPropertyDefinitions"
+  "NonSingleBracedBlock"
   "NonSuppressedTrailingMemberExpressions"
   "ObjectLiteral"
   "PopIndent"
```

Comment:
We should probably also add `PostfixedExpression` to no-cache rules since `AssignmentExpression` and `ExtendedExpression` are no-cache.

Reply:
Good point. I added `PostfixedExpression` and `PostfixedStatement`.